pkg/cmd/pr/shared/display.go

```diff
@@ -8,15 +8,17 @@ import (
 	"github.com/cli/cli/v2/pkg/cmdutil"
 	"github.com/cli/cli/v2/pkg/iostreams"
 	"github.com/cli/cli/v2/utils"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
 )
 
 func StateTitleWithColor(cs *iostreams.ColorScheme, pr api.PullRequest) string {
 	prStateColorFunc := cs.ColorFromString(ColorForPRState(pr))
-
+	c := cases.Title(language.English)
 	if pr.State == "OPEN" && pr.IsDraft {
-		return prStateColorFunc(strings.Title(strings.ToLower("Draft")))
+		return prStateColorFunc(c.String(strings.ToLower("Draft")))
```

Comment:
Does this first convert the constant string `Draft` to lower case and then to title case?